Postinstall fails

I have postinstall script failing. Getting the same error via npm/yarn on different node versions.


```
    ERROR in ./$$_lazy_route_resource lazy
    Module not found: Error: Can't resolve '/Users/rooticalv/Projects/WKL/angular-universal-pwa/node_modules/@angular-universal-serverless/external-module/release/src/externalComponent/external.module.ngfactory.js' in '/Users/rooticalv/Projects/WKL/angular-universal-pwa/$$_lazy_route_resource'
     @ ./$$_lazy_route_resource lazy
     @ ./node_modules/@angular/core/esm5/core.js
     @ dll vendor

    ERROR in ./$$_lazy_route_resource lazy
    Module not found: Error: Can't resolve '/Users/rooticalv/Projects/WKL/angular-universal-pwa/src/app/lazy/lazy.module.ngfactory.js' in '/Users/rooticalv/Projects/WKL/angular-universal-pwa/$$_lazy_route_resource'
     @ ./$$_lazy_route_resource lazy
     @ ./node_modules/@angular/core/esm5/core.js
     @ dll vendor

    ERROR in src/app/windowRef.ts(1,25): error TS2307: Cannot find module '@nguniversal/express-engine/tokens'.
```

--- a/package.json
+++ b/package.json
@@ -47,7 +47,7 @@
     "@angular/router": "5.2.0",
     "@angular/service-worker": "5.2.0",
     "@angularclass/hmr": "^2.1.3",
-    "@nguniversal/express-engine": "^5.0.0-beta.5",
+    "@nguniversal/express-engine": "~5.0.0-beta.6",
     "aws-serverless-express": "3.0.0",
     "body-parser": "^1.17.2",
     "cors": "^2.8.4",

--- a/src/app/windowRef.ts
+++ b/src/app/windowRef.ts
@@ -1,4 +1,4 @@
-import { REQUEST } from '@nguniversal/express-engine/tokens';
+import { REQUEST } from '@nguniversal/express-engine';
 import { Injectable, Inject, PLATFORM_ID, Injector } from '@angular/core';
 import { isPlatformBrowser } from '@angular/common';
 

--- a/webpack/webpack.client.js
+++ b/webpack/webpack.client.js
@@ -7,6 +7,7 @@ module.exports = function (options) {
     entryFile = options.aot? root('./src/main.browser-aot.ts') : root('./src/main.browser.ts');
     return {
         entry: entryFile,
+        node: { fs: 'empty' },
         output: {
             filename: 'client.js'
         },

--- a/webpack/webpack.test.js
+++ b/webpack/webpack.test.js
@@ -108,6 +108,7 @@ module.exports =  {
             crypto: 'empty',
             module: false,
             clearImmediate: false,
-            setImmediate: false
+            setImmediate: false,
+            fs: 'empty'
         }
 }

